sync/config/operator.yaml

```
@@ -29,6 +29,16 @@ archive: https://github.com/tektoncd/operator/tags
 #       header: <dict>         # optional, no header added if not set
 #         See https://www.docsy.dev/docs/adding-content/navigation/#section-menu
 tags:
+  - name: release-v0.78.x
+    displayName: v0.78.x-LTS
+    supportEnds: "2026-12-8" # YYYY-MM-DD
```

nit: date should be zero-padded

```suggestion
    supportEnds: "2026-12-08" # YYYY-MM-DD
```